### PR TITLE
Make visible level of details configurable

### PR DESCRIPTION
**Pull request by sandersky**
#MINOR#

### CODE REVIEW

**Comment by sglanzer** [comment]
Approved

File: addon/components/frost-object-browser.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ export default Ember.Component.extend({
   classNames: ['frost-object-browser'],
   contentHeight: 505,
   detailLevel: 'low',
+  isHighDetailVisible: true,
+  isLowDetailVisible: true,
+  isMediumDetailVisible: false,
```
Comment (job13er) [inline addon/components/frost-object-browser.js]: Curious. Why do we need extra attribute to achieve this? Couldn't you figure out whether the low, medium, and/or high LOD schemas were passed in? Or is the requirement to pass in them all and dynamically hide/show them based on something else?

Comment (sandersky) [inline addon/components/frost-object-browser.js]: There is a UX change to no longer show medium but I figured why not leave it in the open source component as optional. I thought making each level configurable would be nice. For example if I am a consumer and my medium level of details aren't rendering properly I can just toggle a flag until I fix it.

Comment (job13er) [inline addon/components/frost-object-browser.js]: I guess that's fine. But couldn't you just as easily not pass in a medium until you fix it? It seems like having separate flags for visibility when we already pass in details w.r.t. the LOD is redundant and unnecessary unless there's a use-case for providing the schema for the LOD and not displaying it, which doesn't seem likely.

Comment (job13er) [inline addon/components/frost-object-browser.js]: I worry about adding additional configuration options when there isn't a need. If we already have all the information necessary to support the use-cases, I'd prefer not to add additional API surface area.

Comment (sandersky) [inline addon/components/frost-object-browser.js]: Agreed, will revise.

**Comment by job13er** [comment]
:+1:

**Pull request merged**